src/type.ts

```diff
@@ -107,3 +108,10 @@ export interface TagConfig {
 }
 
 export type FeatureIdSource = 'data' | 'data_changed'
+
+export type MapMode = 'Create' | 'Update' | 'Delete';  // FIXME: @dotcs Is this valid?
```

Comment:
Can the reviewer please double check if I got all "map modes" that can happen? Thanks in advance.